README.md

```diff
@@ -125,40 +102,28 @@ Here is a tutorial of the current development features:
 		   Some common rates are:  -r 29.97  -r 25  -r 30  -r 60
 
 
-#################
-    WEBSITES
-#################
-
-http://www.openshot.org/  (PROJECT WEBSITE and BLOG)
-http://launchpad.net/openshot/ (SOURCE CODE, BUG TRACKER, TRANSLATIONS)
-http://freshmeat.net/projects/openshot-video-editor/
-https://sourceforge.net/projects/openshotvideo/
-https://www.ohloh.net/p/openshot-video-editor/
-
-
-
-####################
-    REPORT A BUG
-####################
+## Websites
 
-https://bugs.launchpad.net/openshot
+- http://www.openshot.org/  (Official website and blog)
+- https://github.com/OpenShot/openshot-qt (source code and issue tracker)
+- http://freshmeat.net/projects/openshot-video-editor/
+- https://sourceforge.net/projects/openshotvideo/
+- https://www.ohloh.net/p/openshot-video-editor/
+- http://launchpad.net/openshot/
 
+## Report a bug
 
+You can file an Issue in the OpenShot/openshot-qt project on Github https://github.com/OpenShot/openshot-qt/issues. Make sure to add plenty of detail about the issue and logs if possible.
 
-####################
-    TRANSLATIONS
-####################
+## Translations
 
 https://translations.launchpad.net/openshot
 
+## Ask a question  
 
+You can ask any question you have here https://github.com/OpenShot/openshot-qt/issues.
 
-####################
-   ASK A QUESTION  
-####################
-
-https://answers.launchpad.net/openshot
-
-
-
+## Contributing
 
+If you would like to help reporting issues or commit fixes to the project please see our guidelines:
+https://github.com/OpenShot/openshot-qt/blob/master/CONTRIBUTING.md
```

Comment:
Another option, rather than giving a URL here (and forcing people to the web): Markdown supports relative links, so you could write something like,
```markdown
If you would like to help reporting issues or commit fixes to the project
please see our [contributor guidelines](CONTRIBUTING.md).
```
It'll be a link to the (local) file in the rendered Markdown, and they'll see the filename if they're reading the source.